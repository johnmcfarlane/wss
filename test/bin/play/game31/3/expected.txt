 63  9,1  - XYZ
 50  9,1  | YOWE
 47  9,1  | WYDE
 45  8,1  - OYEZ
 45  9,1  - WIZ
 45  8,1  - YOEZ
 42  9,1  - YEZ
 41  9,1  | WIDE
 41  9,1  | WODE
 39  9,1  - ODZ
 38  9,1  | YODE
 37  9,7  - XI
 32  9,1  | IODE
 30  3,7  - DIOXY
 30  6,6  | EXODY
 29  9,1  | IDEO
 28 10,6  | LOXED
 28  6,2  | OXEYE
 27  4,7  - DIXY
 27  4,7  - DOXY
 27  2,5  - EXODY
 26  6,2  | DOXIE
 26  6,2  | EXDIE
 26  6,6  | EXDIE
 26  6,2  | EXODE
 26  6,6  | EXODE
 26  6,2  | OXIDE
 25  9,6  | EXODY
 25  5,7  - OXY
 23  9,6  | EXODE
 23  3,7  - EXODY
 23  2,5  - OXIDE
 23  2,7  - OXIDE
 23  5,7  - YEX
 23  5,7  - YOX
 22  5,7  - DEX
 22  9,6  | EXDIE
 22  5,6  | WYDE
 22  9,2  | WYE
 22  5,4  | YOWED
 21  5,4  | DEWY
 21  5,4  | DOWY
 21  9,2  | OWE
 21  5,6  | WIED
 21  9,7  - WO
 21  4,7  - YEX
 21  5,4  | YOWIE
 21  4,7  - YOX
 20  4,7  - DEX
 20  2,5  - DIOXY
 20  3,5  - DIXY
 20  5,4  | DOWIE
 20  3,5  - DOXY
 20  9,2  | DYE
 20  6,7  - EX
 20 10,6  | LOWED
 20  6,7  - OX
 20  9,3  | WE
 20  5,6  | WIDE
 20  9,3  | WO
 20  5,6  | WODE
 20  6,7  - XI
 20  5,4  | YOWE
 20  6,2  | YOWIE
 19  4,7  - DEWY
 19  4,7  - DOWY
 19  5,5  | OWED
 19  3,7  - OXIDE
 19  5,6  | WEY
 19  5,6  | WYE
 19  5,4  | YEW
 19  5,4  | YOW
 19  3,7  - YOWED
 18  5,4  | DEW
 18  5,4  | DOW
 18  6,2  | DOWIE
 18  5,7  - EX
 18  5,5  | OWD
 18  5,7  - OX
 18  5,6  | WED
 18  5,6  | WID
 18  9,2  | WOE
 18 10,2  - XI
 18  9,3  | YE
 18  9,3  | YO
 17  9,2  | EDO
 17  9,6  | EX
 17  9,2  | IDE
 17  9,2  | ODE
 17  5,5  | OWE
 17 10,4  | OWLY
 17  5,6  | WEI
 17 10,4  | WILED
 17 10,4  | WILY
 17  5,6  | WOE
 16  9,3  | DE
 16  3,5  - DEWY
 16  9,2  | DIE
 16  9,3  | DO
 16  9,2  | DOE
 16  3,5  - DOWY
 16  2,5  - DOXIE
 16  2,7  - DOXIE
 16  8,3  | EJUSD
 16  5,5  | EW
 16  5,5  | OW
 16  3,5  - OXID
 16  3,7  - OXID
 16  4,5  - OXY
 16  5,6  | WE
 16 10,4  | WELD
 16 10,4  | WILD
 16  5,6  | WO
 16  3,5  - WYDE
 16  3,7  - WYDE
 16  4,7  - YOWE
 15  5,7  - OWD
 15 10,4  | WELI
 15  3,5  - WIED
 15  3,7  - WIED
 15 10,4  | WILE
 15  4,7  - WYDE
 15  5,7  - YEW
 15  5,7  - YOW
 15  2,5  - YOWED
 15  2,7  - YOWED
 14  7,6  | AXED
 14  7,6  | AXOID
 14  5,7  - DEW
 14  5,7  - DOW
 14 10,5  | ELIX
 14  9,3  | EO
 14 10,5  | ILEX
 14  9,3  | IO
 14  9,3  | OE
 14 10,4  | OWL
 14  6,4  | OXEYE
 14  5,7  - WEY
 14  3,5  - WIDE
 14  3,7  - WIDE
 14  3,5  - WODE
 14  3,7  - WODE
 14 10,4  | YELD
 14  3,5  - YOWE
 14  3,7  - YOWE
 13 10,4  | DELY
 13  9,7  - DI
 13  9,7  - DO
 13  8,4  - JOIE
 13  5,7  - OWE
 13  4,7  - OWED
 13  9,3  - WEB
 13  4,7  - WIDE
 13  4,7  - WIED
 13  9,3  - WOB
 13  4,7  - WODE
 13  4,5  - YEW
 13  4,7  - YEW
 13  4,5  - YOW
 13  4,7  - YOW
 13  2,5  - YOWIE
 13  2,7  - YOWIE
 12  4,5  - DEW
 12  4,7  - DEW
 12  5,7  - DEY
 12  4,5  - DOW
 12  4,7  - DOW
 12  2,5  - DOWIE
 12  2,7  - DOWIE
 12  7,5  - DUE
 12  7,5  - DUO
 12  6,7  - EW
 12  9,6  | EYED
 12  9,2  - IWI
 12  6,7  - OW
 12  3,5  - OWED
 12  3,7  - OWED
 12  6,7  - WE
 12  5,7  - WED
 12  9,2  - WEI
 12  4,5  - WEY
 12  6,5  | YEX
 12  3,5  - YODE
 12  3,7  - YODE
 12  4,7  - YODE
 11  7,6  | AWIDE
 11 10,4  | DELI
 11  6,3  | DEWEY
 11  6,5  | DEWEY
 11  6,5  | DEX
 11 10,4  | DILO
 11  8,4  - JE
 11  8,4  - JO
 11 10,6  | LEX
 11 10,6  | LOX
 11 10,4  | OILED
 11 10,4  | OILY
 11  5,6  - WEASELS
 11  6,4  | WEEDY
 11  6,5  | WEEDY
 11  9,3  - YOB
 11  5,7  - YOD
 11  6,3  | YOWED
 10  7,6  | AWED
 10  7,6  | AXE
 10 10,4  | DEL
 10  6,5  | DEWY
 10  4,5  - DEY
 10 10,4  | DIL
 10 10,4  | EILD
 10  5,5  - EW
 10  5,7  - EW
 10  9,6  | EWE
 10  4,7  - IODE
 10 10,6  | LOWY
 10  5,5  - OW
 10  5,7  - OW
 10  4,5  - OWD
 10  4,7  - OWD
 10  6,7  - OY
 10  4,5  - WED
 10  4,7  - WED
 10  5,7  - WEI
 10  4,5  - WID
 10  4,7  - WID
 10  5,7  - WOE
 10  6,3  | WYDE
 10  4,5  - WYE
 10  4,7  - WYE
  9  7,6  | AX
  9  9,3  - DEB
  9  9,3  - DIB
  9  9,3  - DOB
  9  9,7  - EO
  9  9,6  | EW
  9  6,6  | EX
  9  5,7  - IDE
  9 10,6  | LEWD
  9  5,7  - ODE
  9  9,7  - OI
  9  4,5  - YOD
  9  4,7  - YOD
  9  6,3  | YOWE
  8  6,7  - DE
  8  9,2  - DEI
  8  5,7  - DEI
  8  6,7  - DI
  8  5,7  - DOE
  8  4,5  - DYE
  8  4,7  - DYE
  8  6,7  - ED
  8 10,5  | ELOD
  8  9,6  | EYE
  8  3,5  - IDEO
  8  3,7  - IDEO
  8  3,5  - IODE
  8  3,7  - IODE
  8 10,6  | LOWE
  8 10,6  | LOYD
  8 10,6  | LWEI
  8  6,7  - OD
  8 10,4  | OIL
  8  4,5  - OWE
  8  4,7  - OWE
  8  6,4  | OWED
  8  5,5  - OY
  8  6,3  | WEDE
  8  6,5  | WEDE
  8  6,4  | WEED
  8  6,5  | WEED
  8  6,5  | WEY
  8  6,3  | WIDE
  8  6,4  | WIED
  8  6,3  | WODE
  8  4,5  - WOE
  8  4,7  - WOE
  8  6,4  | WYE
  8  6,5  | YEW
  8 10,2  - YI
  7  7,6  | AWD
  7  6,5  | DEW
  7  9,6  | EIDE
  7 10,5  | ELD
  7  6,4  | EYED
  7  6,6  | EYED
  7 10,6  | LOW
  7  5,5  - WE
  7  5,7  - WE
  7  6,5  | WED
  7  6,4  | WEEI
  7  6,5  | WEEI
  7  5,5  - WO
  7  5,7  - WO
  7  6,3  | YEDE
  7  6,5  | YEDE
  7  6,3  | YODE
  6  7,6  | ADY
  6  7,6  | AIDE
  6  7,6  | AWE
  6  6,5  | DEY
  6 10,2  - DI
  6  4,5  - DIE
  6  4,7  - DIE
  6  4,5  - DOE
  6  4,7  - DOE
  6  6,4  | DYE
  6  5,5  - ED
  6  5,7  - ED
  6  4,5  - EDO
  6  9,6  | EDO
  6  4,7  - EDO
  6  6,4  | EWE
  6  6,6  | EWE
  6  5,5  - ID
  6  5,7  - ID
  6  4,5  - IDE
  6  4,7  - IDE
  6 10,5  | ILE
  6 10,6  | LEY
  6 10,6  | LIDE
  6 10,6  | LIDO
  6 10,6  | LIED
  6 10,6  | LODE
  6 10,6  | LOED
  6 10,6  | LOID
  6 10,6  | LOY
  6 10,6  | LYE
  6  5,5  - OD
  6  5,7  - OD
  6  4,5  - ODE
  6  4,7  - ODE
  6  6,7  - OE
  6  6,7  - OI
  6  6,4  | OWE
  6  6,4  | WEE
  6  6,5  | WEE
  6  6,5  | WEI
  6  6,4  | WOE
  6  5,5  - YE
  6  5,7  - YE
  6  5,5  - YO
  6  5,7  - YO
  5  7,6  | AW
  5  7,6  | AYE
  5  5,5  - DE
  5  5,7  - DE
  5  5,5  - DO
  5  5,7  - DO
  5  9,6  | ED
  5  6,3  | EIDE
  5  6,6  | EIDE
  5 10,5  | EL
  5 10,5  - ET
  5  6,6  | EW
  5  6,4  | EYE
  5  6,6  | EYE
  5  6,3  | IDEE
  5  6,4  | IDEE
  5  6,4  | IDEO
  5  6,3  | IODE
  5 10,6  | LED
  5 10,6  | LID
  5 10,6  | LOD
  5 10,4  - WE
  5  6,5  | WE
  5  6,4  | YEE
  5  6,5  | YEE
  4  7,6  | ADO
  4  7,6  | AID
  4  7,6  | AY
  4  6,4  | DEE
  4  6,5  | DEE
  4  6,5  | DEI
  4  6,4  | DIE
  4  6,4  | DOE
  4  6,6  | EDO
  4  5,5  - EO
  4  5,7  - EO
  4  6,4  | IDE
  4  5,5  - IO
  4  5,7  - IO
  4 10,6  | LEI
  4 10,6  | LIE
  4 10,6  | LOE
  4  6,4  | ODE
  4  5,5  - OE
  4  5,7  - OE
  4 10,2  - OI
  4 10,4  - YE
  4  6,5  | YE
  3  7,6  | AD
  3 10,4  - DE
  3  6,5  | DE
  3  6,6  | ED
  3  9,6  | EE
  3  9,6  | EO
  3 10,6  | LI
  3 10,6  | LO
  2  7,6  | AE
  2  7,6  | AI
  2 10,4  - EE
  2  6,5  | EE
  2  6,6  | EE
  2  6,6  | EO
  2 10,4  - OE
  2  6,5  | OE
