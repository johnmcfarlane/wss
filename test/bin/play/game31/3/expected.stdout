 45  8,1  - OYEZ
 45  9,1  - WIZ
 37  9,7  - XI
 30  3,7  - DEOXY
 28 10,6  | LOXED
 28  6,2  | OXEYE
 27  4,7  - DEXY
 27  4,7  - DOXY
 26  6,2  | DEXIE
 26  6,2  | DOXIE
 26  6,2  | OXIDE
 25  5,7  - OXY
 22  9,6  | EXED
 22  5,4  | YOWED
 21  5,4  | DEWY
 21  5,4  | YOWIE
 20  2,5  - DEOXY
 20  4,7  - DEX
 20  3,5  - DEXY
 20  5,4  | DOWIE
 20  3,5  - DOXY
 20 10,6  | LOWED
 20  5,6  | WIDE
 20  9,3  | WO
 20  6,7  - XI
 20  5,4  | YOWE
 20  6,2  | YOWIE
 19  5,5  | OWED
 19  3,7  - OXIDE
 19  5,6  | WYE
 19  5,4  | YEW
 19  5,4  | YOW
 18  5,4  | DEW
 18  5,4  | DOW
 18  6,2  | DOWIE
 18  5,7  - EX
 18  5,7  - OX
 18  5,6  | WED
 18 10,2  - XI
 18  9,3  | YO
 17  9,6  | EX
 17  5,5  | OWE
 17 10,4  | WILED
 17 10,4  | WILY
 17  5,6  | WOE
 16  3,5  - DEWY
 16  9,3  | DO
 16  5,5  | OW
 16  3,5  - OXID
 16  3,7  - OXID
 16  4,5  - OXY
 16  5,6  | WE
 16 10,4  | WELD
 16 10,4  | WILD
 16  5,6  | WO
 15  6,5  | DEOXY
 15 10,4  | WILE
 15  2,5  - YOWED
 15  2,7  - YOWED
 14  7,6  | AXED
 14  6,5  | DEXY
 14 10,5  | ILEX
 14  6,4  | OXEYE
 14 10,4  | YELD
 13  6,5  | DEXIE
 13  9,7  - DO
 13  9,3  - WEB
 13  4,7  - WIDE
 13  4,5  - YEW
 13  4,5  - YOW
 12  4,5  - DEW
 12  4,5  - DOW
 12  6,4  | EXED
 12  6,6  | EXED
 12  9,6  | EYED
 12  3,5  - OWED
 12  3,7  - OWED
 11 10,4  | DELI
 11  6,5  | DEX
 11  8,4  - JO
 11 10,6  | LEX
 11 10,6  | LOX
 11 10,4  | OILED
 11 10,4  | OILY
 11  5,6  - WEASELS
 11  6,4  | WEEDY
 11  6,5  | WEEDY
 11  9,3  - YOB
 11  6,3  | YOWED
 10  7,6  | AWED
 10  7,6  | AXE
 10 10,4  | DEL
 10  6,5  | DEWY
 10  4,5  - DEY
 10  9,6  | EWE
 10  5,5  - OW
 10  4,5  - WED
 10  4,7  - WED
  9  7,6  | AX
  9  9,3  - DEB
  9  9,3  - DIB
  9  6,6  | EX
  9 10,6  | LEWD
  9  5,7  - ODE
  9  4,5  - YOD
  9  4,7  - YOD
  9  6,3  | YOWE
  8  6,7  - DE
  8  9,6  | EYE
  8 10,6  | LOWE
  8 10,6  | LWEI
  8 10,4  | OIL
  8  6,4  | OWED
  8  5,5  - OY
  8  6,4  | WEED
  8  6,5  | WEED
  8  6,3  | WIDE
  8  6,4  | WYE
  8  6,5  | YEW
  7  6,5  | DEW
  7  9,6  | EIDE
  7 10,5  | ELD
  7  6,4  | EYED
  7  6,6  | EYED
  7 10,6  | LOW
  7  6,5  | WED
  7  5,5  - WO
  6  7,6  | AIDE
  6  7,6  | AWE
  6  6,5  | DEY
  6  6,4  | DYE
  6  5,5  - ED
  6  5,7  - ED
  6  6,4  | EWE
  6  6,6  | EWE
  6  5,5  - ID
  6  5,7  - ID
  6 10,6  | LEY
  6 10,6  | LIDO
  6 10,6  | LIED
  6 10,6  | LODE
  6 10,6  | LOID
  6 10,6  | LYE
  6  5,5  - OD
  6  5,7  - OD
  6  6,4  | OWE
  6  6,4  | WEE
  6  6,5  | WEE
  6  6,4  | WOE
  6  5,5  - YO
  5  7,6  | AW
  5  7,6  | AYE
  5  5,5  - DO
  5  9,6  | ED
  5  6,3  | EIDE
  5  6,6  | EIDE
  5 10,5  | EL
  5 10,5  - ET
  5  6,4  | EYE
  5  6,6  | EYE
  5 10,6  | LED
  5 10,6  | LID
  5 10,4  - WE
  5  6,5  | WE
  4  7,6  | ADO
  4  7,6  | AID
  4  7,6  | AY
  4  6,4  | DEE
  4  6,5  | DEE
  4  6,4  | DIE
  4  6,4  | DOE
  4 10,6  | LEI
  4 10,6  | LIE
  4  6,4  | ODE
  4 10,2  - OI
  4 10,4  - YE
  4  6,5  | YE
  3  7,6  | AD
  3 10,4  - DE
  3  6,5  | DE
  3  6,6  | ED
  3 10,6  | LI
  3 10,6  | LO
  2  7,6  | AE
  2  7,6  | AI
  2 10,4  - OE
  2  6,5  | OE
